scripts/initiate_ritual.py

```diff
@@ -133,29 +154,52 @@ def cli(
             duration -= elapsed
 
     # Get the staking providers in the ritual cohort
-    if handpicked:
-        providers = sorted(line.lower().strip() for line in handpicked)
-        if not providers:
-            raise ValueError(f"No staking providers found in the handpicked file {handpicked.name}")
+    if heartbeat:
+        taco_application = registry.get_contract(domain=domain, contract_name="TACoChildApplication")
+        cohorts = get_heartbeat_cohorts(taco_application=taco_application)
+        click.echo(f"Number of rituals: {len(cohorts)}")
+        click.confirm(text="Are you sure you want to initiate these rituals?", abort=True)
```

Comment:
Does this prompt have to be manually confirmed? If so, is this just temporary for now?

Reply:
Yes - just temporary for now while we are manually testing 👍🏻

Reply:
✔️